fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -127,6 +130,11 @@ def self.available_options
                                            UI.user_error!("Please pass a valid platform. Use one of the following: #{available_platforms.join(', ')}") unless available_platforms.map(&:downcase).include?(platform.downcase)
                                          end
                                        end),
+          FastlaneCore::ConfigItem.new(key: :cache_builds,
+                                       env_name: "FL_CARTHAGE_CACHE_BUILDS",
+                                       description: "By default Carthage will rebuild a dependency regardless of whether it\'s the same resolved version as before. Passing the --cache-builds will cause carthage to avoid rebuilding a dependency if it can",
+                                       is_string: false,
+                                       optional: true),
```

Comment:
@giginet i think `optional: true` in combination with `is_string: false`  has the same effect, and is used accross the whole codebase. @KrauseFx  am i wrong?